src/ExtensionsSample/Program.cs

```diff
@@ -65,7 +66,8 @@ public static void Main(string[] args)
                 typeof(SendGridSamples),
                 typeof(TableSamples),
                 typeof(TimerSamples),
-                typeof(WebHookSamples));
+                typeof(WebHookSamples),
+                typeof(NotificationHubSamples));
```

Comment:
Remove NotificationHubSamples from here -- we shouldn't have that on by default since it requires extra setup.